src/hb-ot-os2-table.hh

```diff
@@ -49,6 +49,51 @@ struct os2
     return_trace (c->check_struct (this));
   }
 
+  inline hb_bool_t subset (hb_subset_plan_t *plan) const
+  {
+    hb_blob_t *os2_blob = OT::Sanitizer<OT::os2>().sanitize (hb_face_reference_table (plan->source, HB_OT_TAG_os2));
```

Comment:
So, these subset functions seem to be static, no? You are reloading the table...  If yes, mark them static for now. `static HB_INTERNAL` I think.  At some point we'll change that.